phpcs.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<ruleset name="WikibaseDataModelServices">
+	<rule ref="vendor/wikibase/wikibase-codesniffer/Wikibase" />
+
+	<rule ref="Generic.Files.LineLength">
+		<properties>
+			<property name="lineLimit" value="120" />
+		</properties>
+	</rule>
+
+	<rule ref="Generic.Arrays.DisallowLongArraySyntax.Found">
+		<exclude name="Generic.Arrays.DisallowLongArraySyntax.Found" />
```

Comment:
I didn't count how many issues this found but there were several. I've first raised the limit to 120 and there were only two problems so I chopped those down. I can make it use the default (100 chars AFAIK) but then I am going to be completely ignoring comments like "The diff looks horrible" :)